src/integrationTest/resources/wiremock/responses/about-to-submit-system-regenerate-court-orders.json

```diff
@@ -23,6 +23,18 @@
       "documentType": "conditionalOrderGranted"
     },
     "documentsGenerated": [
+      {
+        "id": "${json-unit.any-string}",
+        "value": {
+          "documentLink": {
+            "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3",
+            "document_filename": "${json-unit.any-string}",
+            "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3/binary"
+          },
+          "documentFileName": "${json-unit.any-string}",
+          "documentType": "certificateOfEntitlement"
+        }
+      },
```

Comment:
This is the bit I am unsure of: previously the certificateOfEntitlement was not showing up in documentsGenerated but there is a value here under coCertificateOfEntitlementDocument so if someone could clarify where this one is meant to show up that'd be great

Reply:
Faisal has explained that the cert of entitlrement is a bit weird in that it doesn't sit in generatedDocs usually but that the letter print prints it from there so when regenerated, it gets created in generatedDoc, then copied over to it's own casedata section and then needs to be cleared out but not until after letter print